server/utils/datetime_utils.py

```
@@ -0,0 +1,162 @@
+from typing import Union
+
+# Register NetAlertX directories
+INSTALL_PATH="/app"
+sys.path.extend([f"{INSTALL_PATH}/front/plugins", f"{INSTALL_PATH}/server"])
+
+
+# Register NetAlertX directories
+INSTALL_PATH="/app"
+
+import conf
+from const import *
```

Ruff (0.14.3)

`from const import *` used; unable to detect undefined names

(F403)